packages/examples/require-template-literals/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you remove the example? The new tests are enough

Reply:
Sure thing.
I will make a new commit with the directory removed from the source.

Reply:
(I appreciate your diligence with the commits and their descriptions, but we [squash all PRs anyway](https://github.com/parcel-bundler/parcel/commits/v2), so I'm afraid they will not actually be used. Might save you some time here 😄 )

Reply:
Thanks for the heads up! 🙂 
I usually put descriptive commit messages to describe the intent behind a single commit and for an easier git flow management (reverts, chery-picks, etc), thought it's more of a personal preference and I thought it might save you some time during the code review.

I started contributing to OSS recently and I'm still new, so I really do appreciate the feedbacks and of course your time and effort you put in to reviewing the PR.
I do always read contribution guidelines for each project, but it never occured to me to check the release flow; I might add it to my workflow to save myself some time 😄 .